packages/react-router/lib/dom/ssr/routes-test-stub.tsx

```diff
@@ -100,10 +100,10 @@ export function createRoutesStub(
     future,
   }: RoutesTestStubProps) {
     let routerRef = React.useRef<ReturnType<typeof createMemoryRouter>>();
-    let remixContextRef = React.useRef<FrameworkContextObject>();
+    let reactRouterContextRef = React.useRef<FrameworkContextObject>();
```

Comment:
Let's keep this in line with the type

```suggestion
    let frameworkContextRef = React.useRef<FrameworkContextObject>();
```